### ISSUE TITLE
Support a printing option to split function arguments onto multiple lines when the line is long

### ISSUE THREAD

**Issue by dalongi** [benjamn/recast#205]
I recently used jscodeshift to add an additional argument to many function calls. Even when specifying wrapColumns, the function was still printed as one very long line instead of split over many lines.

It would be nice to have a way to have wrapColumns respected for functions with many arguments.

CC @cpojer

**Comment by benjamn** [comment]
Just to make sure: is there any chance this could be explained by the accidental use of `options.wrapColumns` instead of [`options.wrapColumn`](https://github.com/benjamn/recast/blob/9fd7bdaadb161c44f5d350a41db3761f247f355f/lib/options.js#L24)?

**Comment by dalongi** [comment]
Yes, sorry, that was just a typo in the description. Live example:
http://felix-kling.de/esprima_ast_explorer/#/Ba4MGXTGGN/2

**Comment by dalongi** [comment]
I looked into this a little bit to see if there is a simple fix. I figured out the first one, but don't know how to approach the 2nd.
- If any single argument prints to > 1 line, it only counts the chars on the first line
- It only compares the string length of arguments to the wrapColumn, but doesn't include anything else on the line (e.g. callee name, parens)
